docs/user-guide/rules/highest-available-document-mode.md

```diff
@@ -283,6 +283,77 @@ Note that:
 
 </details>
 
+<details>
+<summary>How to configure IIS</summary>
+
+### Adding the `X-UA-Compatible` header
+
+To add the `X-UA-Compatible` header you can use a [`URL rewrite` rule][url rewrite]
+that matches the `text/html` content-type of a response and then adds it:
```

Comment:
`application/xhtml+xml` is also commonly used.

Reply:
content-type => media type

Reply:
> content-type => media type

I'm matching the `content-type` header.

Reply:
> `application/xhtml+xml` is also commonly used.

I'll make a note bellow replacing. Adding rules and conditions has a perf impact so user should try to make the rewrite rules as lean as possible.